tutorial/file-formats.md

```diff
@@ -0,0 +1,40 @@
+# File Formats
+File formats are specific ways to store data that can be read and written using standardised methods.
+
+## Proposal Requirements
+Every format must specify all of the following:
+
+### File Extension
+Example: `.lua`
+
+If the file format is new it must only be given extension; there's no point have 3 different extensions for the same
+file.
+
+If the file format is an existing format *and it already* uses multiple extensions then you should list all of the
+commonly used ones.
+
+### MIME
+Example: `text/lua`, `image/sketch`, `silica/theme`
+
+Every file format must be given ***one*** MIME. If more than one already exists pick **one**.
```

Comment:
Perhaps "exactly one" instead of ***one***? Precise language can't get lost through format stripping from copy-pasting the way formatting emphasis can.